lib/js/atoms/Badge/Badge.stories.mdx

```diff
@@ -0,0 +1,27 @@
+import { Meta, Story, Props } from '@storybook/addon-docs/blocks';
+import Badge from './Badge'; import { BADGE_COLORS } from './Badge.consts';
+
+<Meta title='atoms/Badge' component={Badge} />
+
+# Badge
+
+<Story name='interactive'
+	   args={{
+		   label: 'Komentarz zespołu',
+		   iconUrl: 'https://lek.wiecejnizlek.pl/images/lek/logo-badge.svg',
```

Comment:
There is no `iconUrl` prop in `Badge` component.

Reply:
And I don't pass this value to prop. I have to present this component for DR purses.

Reply:
Ah, I see now 👍 Expect some questions, though, because having a controlable argument suggests there is a prop like that.